### PR TITLE
[TM-4582] Cycle Position Search Card

### PR DESCRIPTION
**Pull request by jlam-mpc**
Feature:
- Added Cycle Position Card using reusable component 

[TM-4582](https://metaphase.atlassian.net/browse/TM-4582)

[TM-4582]: https://metaphase.atlassian.net/browse/TM-4582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/CycleManagement/CycleManagement.jsx [hunk 1]
```diff
@@ -117,12 +118,12 @@ const CycleManagement = () => {
...
-      (
-        <div className="cycle-management-page">
+    genericFiltersIsLoading ?
+      <Spinner type="bureau-filters" size="small" /> :
+      <>
+        <div className="bureau-page cycle-management-page">
           <div className="usa-grid-full cm-upper-section">
-
-            <ProfileSectionTitle title="Cycle Search" icon="keyboard-o" />
+            <ProfileSectionTitle title="Cycle Search" icon="keyboard-o" className="xl-icon" />
```
Comment (elizabeth-jimenez) [inline src/Components/CycleManagement/CycleManagement.jsx:126]: `ProfileSectionTitle` doesn't receive a className prop

File: src/Components/PublishablePositionCard/PublishablePositionCard.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const PublishablePositionCard = ({ data }) => {
     },
     bodyPrimary: {
       'Bureau': getResult(pos, 'bureau_short_desc') || NO_BUREAU,
-      'Location': getPostName(get(pos, 'post') || NO_POST),
```
Comment (elizabeth-jimenez) [inline src/Components/PublishablePositionCard/PublishablePositionCard.jsx:30]: nice catch!

File: src/Components/CyclePositionCard/CyclePositionCard.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import { get } from 'lodash';
+import { getDifferentials, getResult } from 'Components/ResultsCard/ResultsCard';
```
Comment (elizabeth-jimenez) [inline src/Components/CyclePositionCard/CyclePositionCard.jsx:2]: Let's move these functions into Utilities and use it from there - you don't have to worry about transitioning old components that have already done it this way, just for the new ones

File: src/Components/CyclePositionCard/CyclePositionCard.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      'Grade': getResult(pos, 'grade') || NO_GRADE,
+      'Status': getResult(pos, 'status') || NO_STATUS,
+      'Language': <LanguageList languages={getResult(pos, 'languages', [])} propToUse="representation" />,
+    },
+    bodySecondary: {
+      '': <CheckBox id="deto" label="DETO" value disabled />,
+      'Bid Cycle': getResult(pos, 'latest_bidcycle.name', 'None Listed'),
+      'Cycle Position': '---',
+      'Tour of Duty': getResult(pos, 'post.tour_of_duty') || NO_TOUR_OF_DUTY,
+      'Incumbent TED': getResult(data, 'ted') || NO_DATE,
```
Comment (elizabeth-jimenez) [inline src/Components/CyclePositionCard/CyclePositionCard.jsx:42]: You can use `NO_TOUR_END_DATE` here, if you prefer

**Comment by elizabeth-jimenez** [comment]
Approve Pending Feedback: Just copying those functions over to Utilities file and calling them from there instead

File: src/Components/CycleManagement/CycleManagement.jsx (hunk 1 above)
Comment (jlam-mpc) [reply src/Components/CycleManagement/CycleManagement.jsx:126]: Okay I'll style that outside of the ProfileSectionTitle component in my next push

Comment (jlam-mpc) [reply src/Components/CycleManagement/CycleManagement.jsx:126]: Oh Elizabeth it's something I added by accident in this branch while I was standardizing the Bureau pages is it okay to leave it for when the other PR gets pushed?

Comment (elizabeth-jimenez) [reply src/Components/CycleManagement/CycleManagement.jsx:126]: Yes, that's fine!

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 5]
```diff
@@ -11,75 +10,24 @@ import Favorite from '../../Containers/Favorite';
...
 import HoverDescription from './HoverDescription';
-import OBCUrl from '../OBCUrl';
 import BidListButton from '../../Containers/BidListButton';
 import bannerImg from '../../assets/svg/card-flag.svg';
 
-import { formatDate, getBidStatisticsObject, getDifferentialPercentage, getPostName,
-  propOrDefault, shortenString } from '../../utilities';
+import {
+  getBidStatisticsObject, getBidStatsToUse, getDifferentials, getPostName, getPostNameText,
+  getResult, propOrDefault, renderBidCount, renderBidCountMobile, shortenString,
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCard/ResultsCard.jsx:22]: 🙀 above and beyond 🥲

**Pull request merged**